packages/react/src/themes/teams-high-contrast/components/Button/buttonVariables.ts

```diff
@@ -3,6 +3,8 @@ import { ButtonVariables } from '../../../teams/components/Button/buttonVariable
 export interface ButtonHighContrastVariables {
   backgroundColorFocus: string
   primaryBackgroundColorFocus: string
+  textColorIconOnly: string
```

Comment:
Is it a good idea to have variables that are only used in high contrast theme? Seems like we should actually remove this whole section...?